gh-actions/retest/package.json

```diff
@@ -46,6 +46,6 @@
     "@octokit/rest": "^20.0.1",
     "axios": "^1.6.8",
     "fetch-mock": "^9.11.0",
-    "node-fetch": "^2.7.0"
+    "node-fetch": "^3.0.0"
```

Comment:
i think this breaks the limited test/s in retest - i couldnt get it to work with 3+ locally

Reply:
yep 8/

Reply:
i think yes - i dont recall exact error it resolved, but i think that was the gist of it

Reply:
(not sure wrt full jest solution)